util/settings/settings.go

```diff
@@ -282,53 +344,6 @@ func (mgr *SettingsManager) GetSettings() (*ArgoCDSettings, error) {
 	return &settings, nil
 }
 
-// MigrateLegacyRepoSettings migrates legacy (v0.10 and below) repo secrets into the v0.11 configmap
-func (mgr *SettingsManager) MigrateLegacyRepoSettings(settings *ArgoCDSettings) error {
```

Comment:
did you mean to delete this func?

Reply:
Yes. This was mostly required for argocd with legacy settings deployed at Intuit.